packages/graphback-datasync/src/util.ts

```diff
@@ -38,3 +33,9 @@ export class ConflictError extends Error {
     this.conflictInfo = stateMap;
   }
 }
+
+export const DataSyncFieldNames = {
+  version: '_version',
```

Comment:
I though we will use double underscores same as graphql introspection format.

Reply:
Sure, we can go with that

Reply:
All three are using double underscores now